cheroot/server.py

```diff
@@ -1866,8 +1868,27 @@ def prepare(self):  # noqa: C901  # FIXME
         self.ready = True
         self._start_time = time.time()
 
+    def _serve_unservicable(self):
+        """Serve connections we can't handle a 503."""
+        while self.ready:
+            conn = self._unservicable_conns.get()
+            if conn is None:
+                return
```

Comment:
I think we should call `shutdown()` here (under Python 3.13+).

Reply:
I just learned about https://docs.python.org/3/library/queue.html#queue.Queue.shutdown and thought that it could perhaps be incorporated. I haven't used it myself, so I'm asking if there's a place for calling this new method.

Reply:
Ahh, yes that might be useful.